docs/references/http_api_reference.md

```diff
@@ -60,15 +60,15 @@ curl --request DELETE \
 
 #### Response
 
-- 200 success
+- 200 Success.
 
 ```
 {
     "error_code": 0
 }
 ```
 
-- 500 Internal Service Error
+- 500 Internal service error.
```

Comment:
Why in some cases 500 mean error only?

Reply:
This is HTTP status code means error happens in server.

Reply:
500 Error.